### PR TITLE
[e2e] Push state and navigate to integration edit

### PR DESCRIPTION
**Pull request by jludvice**

### CODE REVIEW

**Comment by jludvice** [comment]
Contains
* fix for login page (better wait to avoid situations where tests think we're already authenticated but we weren't redirected to github yet)
* move integration steps from narative to `integration.steps.ts`
* steps to navigate to saved integration

@dsimansk I'd like to merge this because @lvydra is going to reuse parts of this work in his tests

File: e2e/integrations/integration.steps.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  @then(/^she is presented with a visual integration editor for "([^"]*)"$/)
+  public editorOpenedFor(integrationName: string): P<any> {
+    return this.editorOpened().then(() => {
+      // ensure we're on editor page and then check integration name
+        const page = new IntegrationEditPage();
+        return expect(page.flowViewComponent().getIntegrationName(), `editor must display integration name ${integrationName}`)
+          .to.eventually.be.equal(integrationName);
+      }
+    ).catch(e => P.reject(e));
+  }
```
Comment (jludvice) [inline e2e/integrations/integration.steps.ts]: Can anybody see where is the missing comma supposed to be?
```
yarn ng v0.23.2
$ "/workspace/node_modules/.bin/ng" lint

e2e/integrations/integration.steps.ts[44, 7]: Missing trailing comma
```

Comment (gashcrumb) [inline e2e/integrations/integration.steps.ts]: I'm not seeing where it's missing either.  Maybe play around with the code formatting a bit?  Could be that lone `}` is confusing the parser somehow, or maybe there's a missing `}` elsewhere in the file?

**Comment by dsimansk** [comment]
@jludvice thanks for finishing this in especially in meeting busy day 👍 and for even more improved login flow :)

File: e2e/integrations/integration.steps.ts (hunk 1 above)
Comment (dsimansk) [inline e2e/integrations/integration.steps.ts]: @jludvice more the trailing bracket together with curly bracket, it did the trick for me.

```
     })
.catch(e => P.reject(e));
// or
    }).catch(e => P.reject(e));

```

**Comment by jludvice** [comment]
@dsimansk please review this

**Pull request merged**